plz.el

```diff
@@ -320,6 +320,10 @@ NOTE: In v0.8 of `plz', only one error will be signaled:
 to update their code while using v0.7 (i.e. any `condition-case'
 forms should now handle only `plz-error', not the other two).
 
+DURING is an optional callback function called each time a part
+of the response body is arriving.  It is called with one
+argument, the partial raw response.
```

Comment:
Here, and in the similar documentation below, can you make clear if the partial raw response is the cumulative response or the new response since the last callback?  Also, you may want to make it clear whether it is expected to get all of the content eventually or not.